server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryErrorTest.java

```diff
@@ -370,6 +382,57 @@ public void remoteDeliveryShouldNotDuplicateContentWhenSendPartial() throws Exce
                 "]");
     }
 
+    @Ignore("JAMES-2097 Using full recipients for following MX iteration when partial fails on delivering")
+    @Test
+    public void remoteDeliveryShouldNotDuplicateContentWhenSendPartialWhenFailover() throws Exception {
+        ImmutableList<InetAddress> addresses = ImmutableList.of(InetAddress.getByName(mockSmtp.getContainerIp()));
+        ImmutableList<String> mxs = ImmutableList.of(mockSmtp.getContainerIp(), mockSmtp2.getContainerIp());
+        ImmutableList<String> txtRecords = ImmutableList.of();
+
+        inMemoryDNSService.registerRecord(ANOTHER_DOMAIN, addresses, mxs, txtRecords)
+            .registerMxRecord(mockSmtp.getContainerIp(), mockSmtp.getContainerIp())
+            .registerMxRecord(mockSmtp2.getContainerIp(), mockSmtp2.getContainerIp());
+
+        with()
```

Comment:
will be more clear if you declare a rest client for `mockstmp` and `mockstmp2` like:
```java
mockSmtp1Client = requestSpecification(mockSmtp1.getContainerIp());
mockSmtp2Client = requestSpecification(mockSmtp2.getContainerIp());

with(mockSmtp1Client)
            .body(ALWAYS_PARTIAL_RCPT_421_BEHAVIOR)
            .put("/smtpBehaviors");
```

Reply:
`with` do not take arguments

`given` does